components/delegations/index.ts

```diff
@@ -0,0 +1,13 @@
+export * from './modals/Approval';
+export * from './modals/Confirm';
+export * from './modals/DelegateModal';
+export * from './modals/InputDelegateMkr';
+export * from './modals/TransactionInProgress';
+export * from './modals/TxDisplay';
+export * from './modals/TxFinal';
+export * from './modals/UndelegateModal';
+export * from './DelegateCard';
+export * from './DelegateContractExpiration';
+export * from './DelegateDetail';
+export * from './DelegateLastVoted';
+export * from './DelegatePicture';
```

Comment:
Not totally opposed to this, but it's not my favorite :P Sometimes I find it makes debugging harder. But if we want to adopt this style maybe we should add it to our code standards file to try to achieve uniformity across our apps.

Reply:
Initially made this update because the delegate components were a mix of named and default exports. I personally prefer this pattern because it makes imports very clear and can be done on one line. 

But I'm open to other patterns. The bigger thing to me is consistency throughout the app, so we should talk about it for sure.

Reply:
I like named exports because they are easier to search through the codebase, and it does not leave unnamed functions. 

About the `export * from x` i don't have a strong opinion but sometimes i had circular dependencies because of this. At the end we can find easily any file with the autocomplete. So i think i will vote for separate files with named exports but not having index.ts files that do a aggregation